src/api/currencies.ts

```diff
@@ -13,6 +13,20 @@ export const loadUserCurrencies = async () => {
   return result.map(item => new UserCurrencyRecord(item));
 };
 
+// export const deleteCustomRate = (baseCode: string, quoteCode: string) => {
+//   api.delete('/user/currency/rates', pairs:{baseCode, quoteCode});
+// }
+export const deleteCustomRate = async (
+  pairs: {
+    baseCode: string;
+    quoteCode: string;
+  }[],
+) => {
+  const result = await api.delete('/user/currency/rates', { pairs });
+
+  return result;
+};
```

Comment:
You don't need to await this request inside this function, it's a useless operation. Just return the promise. You need to await only if there's some data normalization inside or other response-oriented stuff.